### ISSUE TITLE
Syntax Error : max msvc building failure

### ISSUE THREAD

**Issue by C-rror** [grrrr/flext#34]
Hi,

**TL;DR : I face a syntax error when building _flext_ under windows, although I successfuly built it under OSX.**

I am trying to build flext for _Max 7_ with _msvc_.
After configuring `config-win-max-msvc.txt`, the second run of `build max msvc build` fails as follows : 
```
------------------------------------------------

D:\Downloads\flext-master>nmake -f "D:\Downloads\flext-master\buildsys\nmake.mak" PLATFORM=win RTSYS=max COMPILER=msvc BUILDPATH=D:\Downloads\flext-master\buildsys\

Microsoft (R) Program Maintenance Utility Version 14.12.25835.0
Copyright (C) Microsoft Corporation. Tous droits réservés.

        "D:\Softwares\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.12.25827\bin\HostX86\x86\nmake.exe" /NOLOGO /f D:\Downloads\flext-master\buildsys\nmake-sub.mak  PLATFORM=win RTSYS=max COMPILER=msvc  BUILDPATH=D:\Downloads\flext-master\buildsys\ PKGINFO=package.txt BUILDCLASS=flext  USRCONFIG=config.txt USRMAKE=build\nmake-win-msvc.inc TARGETMODE=release TARGETTYPE=single _build_
        cl /c   /DNDEBUG /Ox /DWINVER=0x0501 /D_WIN32_WINNT=0x501 /DWIN_VERSION /DWIN_EXT_VERSION /YXflext.h /Fpmax-msvc\release-single\precompiled.pch /LD /EHsc /GR /Oi /nologo /D_CRT_SECURE_NO_DEPRECATE /DWIN32 /D_WINDOWS /MD  /DFLEXT_USE_SIMD /DFLEXT_EXPORTS /DFLEXT_SYS=1  /I"D:\Max_7\Packages\max-sdk-7.3.3\source\c74support"\max-includes /I"D:\Max_7\Packages\max-sdk-7.3.3\source\c74support"\jit-includes /I"D:\Max_7\Packages\max-sdk-7.3.3\source\c74support"\msp-includes source\flbase.cpp /Fomax-msvc\release-single\flbase.obj
cl : Ligne de commande warning D9002 : option '/YXflext.h' inconnue ignorée
flbase.cpp
d:\downloads\flext-master\source\flclass.h(1119): error C2061: erreur de syntaxe : identificateur 'Point'
NMAKE : fatal error U1077: '"D:\Softwares\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.12.25827\bin\HostX86\x86\cl.EXE"' : code retour '0x2'
Stop.
NMAKE : fatal error U1077: '"D:\Softwares\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.12.25827\bin\HostX86\x86\nmake.exe"' : code retour '0x2'
Stop.

D:\Downloads\flext-master>goto end
```
Sorry for that French command line tool (still cannot figure how to change its language).

So the error here is : 
 ```
d:\downloads\flext-master\source\flclass.h(1119): error C2061: erreur de syntaxe : identificateur 'Point'
NMAKE : fatal error U1077:
```
Which translates as `syntax error : indentifier 'Point'`.
Identifier 'Point is only found in `flclass.h` in this bloc (ll.1115-1119) : 
```
1115 #if FLEXT_SYS == FLEXT_SYS_MAX
1116    char **indesc,**outdesc;
1117
1118    static void cb_assist(flext_hdr *c,void *b,long msg,long arg,char *s);
1119        static void cb_click (flext_hdr *c, Point pt, short mods);
```

I can't find how to work around this error, since it lays in fext's source's headers...
**PLUS** I successfuly built _flext_ **and** _flext-based_ externals under OSX (using GCC) this week.
How can a syntax error be plateform-dependent ?

I also note the `warning D9002 : unknown '/YXflext.h' option ignored`...

If anyone has any insight on this...

TS

**Comment by grrrr** [comment]
Hi, in this case, the syntax error means something like "identifier not found". The `point` type was defined in older versions of the Max SDK, but is no longer. The functionality of the assist callback needs to be reviewed.

**Comment by C-rror** [comment]
> Hi, in this case, the syntax error means something like "identifier not found". The `point` type was defined in older versions of the Max SDK, but is no longer. The functionality of the assist callback needs to be reviewed.

Thank you for your reply !
Sadly, this is way beyond my skills...

Are you still working on flext at all ? Or shall I look for another solution :)
